src/analysis/retail/monk/mistweaver/modules/features/RisingMistBreakdown.tsx

```diff
@@ -0,0 +1,282 @@
+    super(options);
+    this.active = this.selectedCombatant.hasTalent(talents.RISING_MIST_TALENT);
+  }
+  protected risingMist!: RisingMist;
+
+  sortedRisingMistItems() {
+    this.risingMistItems = [
+      {
+        spell: SPELLS.RISING_MIST_HEAL,
+        amount: this.risingMist.directHealing,
+        color: SPELL_COLORS.DANCING_MIST,
+        tooltip: this.risingMistDirectTooltip(),
+      },
+      {
+        spell: SPELLS.RENEWING_MIST_HEAL,
+        amount: this.risingMist.renewingMistExtensionHealing,
+        color: SPELL_COLORS.RENEWING_MIST,
+        tooltip: this.renewingMistTooltip(),
+      },
+      {
+        //enveloping mist extension healing
+        spell: talents.ENVELOPING_MIST_TALENT,
+        amount:
+          this.risingMist.envMistyPeaksExtensionHealing +
+          this.risingMist.envHardcastExtensionHealing,
+        color: SPELL_COLORS.ENVELOPING_MIST,
+        tooltip: this.envelopingMistTooltip(),
+        subSpecs: [
+          {
+            //bonus healing from healing bonus
+            spell: talents.ENVELOPING_MIST_TALENT,
+            amount: this.risingMist.extraEnvBonusHealing,
+            color: SPELL_COLORS.BLACKOUT_KICK_TOTM,
+            tooltip: this.envelopingMistBonusHealingTooltip(),
+          },
+        ],
+      },
+      {
+        spell: SPELLS.VIVIFY,
+        amount: this.risingMist.extraVivHealing,
+        color: SPELL_COLORS.VIVIFY,
+        tooltip: this.vivifyTooltip(),
+      },
+      {
+        //essence font extension healing
+        spell: talents.ESSENCE_FONT_TALENT,
+        amount: this.risingMist.essenceFontExtensionHealing,
+        color: SPELL_COLORS.ESSENCE_FONT,
+        tooltip: this.essenceFontTooltip(),
+        subSpecs: [
+          {
+            //additional mastery hits from extended EF hots
+            spell: SPELLS.GUSTS_OF_MISTS,
+            amount: this.risingMist.extraMasteryhealing,
+            color: SPELL_COLORS.GUSTS_OF_MISTS,
+            tooltip: this.gustsOfMistsTooltip(),
+          },
+        ],
+      },
+    ];
+    const sortedRisingMistItems = this.risingMistItems.sort(
+      (a, b) =>
+        b.amount +
+        (b.subSpecs ? b.subSpecs?.reduce((sum, subSpec) => sum + (subSpec?.amount || 0), 0) : 0) -
+        a.amount +
+        (a.subSpecs ? a.subSpecs?.reduce((sum, subSpec) => sum + (subSpec?.amount || 0), 0) : 0),
+    );
+
+    //determine scale factor for chart based on data
+    const scaleFactor = sortedRisingMistItems.reduce(
+      (factor, item) =>
+        factor <
+        this.risingMist.totalHealing /
+          (item.amount +
+            (item.subSpecs
+              ? item.subSpecs?.reduce((sum, subSpec) => sum + (subSpec?.amount || 0), 0)
+              : 0))
+          ? factor
+          : this.risingMist.totalHealing /
+            (item.amount +
+              (item.subSpecs
+                ? item.subSpecs?.reduce((sum, subSpec) => sum + (subSpec?.amount || 0), 0)
+                : 0)),
+      100,
+    );
```

Comment:
The explanation of the param is declared in definition for TalentAggregateStatistic but I can add a comment on the calc in RMBreakdown since its kinda gnarly
![image](https://user-images.githubusercontent.com/26779541/221347894-438e25bf-4c8c-4618-8617-88610349d03a.png)